src/Microsoft.PowerShell.SDK/Microsoft.PowerShell.SDK.csproj

```diff
@@ -17,87 +16,18 @@
-    <PackageReference Include="System.Security.Cryptography.Csp" Version="4.3.0" />
-    <PackageReference Include="System.Security.Cryptography.Pkcs" Version="4.3.0" />
-    <PackageReference Include="System.Security.Principal" Version="4.3.0" />
-    <PackageReference Include="System.Security.Principal.Windows" Version="4.3.0" />
-    <PackageReference Include="System.Security.SecureString" Version="4.3.0" />
-    <PackageReference Include="System.ServiceModel.Duplex" Version="4.3.0" />
-    <PackageReference Include="System.ServiceModel.Http" Version="4.3.0" />
-    <PackageReference Include="System.ServiceModel.NetTcp" Version="4.3.0" />
-    <PackageReference Include="System.ServiceModel.Primitives" Version="4.3.0" />
-    <PackageReference Include="System.ServiceModel.Security" Version="4.3.0" />
-    <PackageReference Include="System.ServiceProcess.ServiceController" Version="4.3.0" />
-    <PackageReference Include="System.Text.Encodings.Web" Version="4.3.0" />
-    <PackageReference Include="System.Threading.AccessControl" Version="4.3.0" />
-    <PackageReference Include="System.Threading.Overlapped" Version="4.3.0" />
-    <PackageReference Include="System.Threading.Tasks.Dataflow" Version="4.7.0" />
-    <PackageReference Include="System.Threading.Tasks.Parallel" Version="4.3.0" />
-    <PackageReference Include="System.Threading.Thread" Version="4.3.0" />
-    <PackageReference Include="System.Threading.ThreadPool" Version="4.3.0" />
-    <PackageReference Include="System.Xml.XmlDocument" Version="4.3.0" />
-    <PackageReference Include="System.Xml.XmlSerializer" Version="4.3.0" />
-    <PackageReference Include="System.Xml.XPath" Version="4.3.0" />
-    <PackageReference Include="System.Xml.XPath.XDocument" Version="4.3.0" />
-    <PackageReference Include="System.Xml.XPath.XmlDocument" Version="4.3.0" />
-    <PackageReference Include="System.Private.ServiceModel" Version="4.3.0" />
+  <ItemGroup>
+    <PackageReference Include="System.Data.SqlClient" Version="4.4.0-preview1-25204-02" />
+    <PackageReference Include="System.IO.Packaging" Version="4.4.0-preview1-25204-02" />
+    <PackageReference Include="System.Net.Http.WinHttpHandler" Version="4.4.0-preview1-25204-02" />
+    <PackageReference Include="System.ServiceModel.Duplex" Version="4.4.0-beta-25205-01" />
+    <PackageReference Include="System.ServiceModel.Http" Version="4.4.0-beta-25205-01" />
+    <PackageReference Include="System.ServiceModel.NetTcp" Version="4.4.0-beta-25205-01" />
+    <PackageReference Include="System.ServiceModel.Primitives" Version="4.4.0-beta-25205-01" />
+    <PackageReference Include="System.ServiceModel.Security" Version="4.4.0-beta-25205-01" />
+    <PackageReference Include="System.Text.Encodings.Web" Version="4.4.0-preview1-25204-02" />
+    <PackageReference Include="System.Threading.AccessControl" Version="4.4.0-preview1-25204-02" />
+    <PackageReference Include="System.Private.ServiceModel" Version="4.4.0-beta-25205-01" />
```

Comment:
`netcoreapp2.0` now includes most of those dependencies by default, and that's why many get removed from SDK.csproj. Now I'm inclined to use `SDK.csproj` as:
1. meta project that references other projects
2. meta project that references those packages that are not directly depended by powershell projects.